_extensions/gdscript.py

```diff
@@ -9,45 +9,37 @@
     :license: MIT.
 
     modified by Daniel J. Ramirez <djrmuv@gmail.com> based on the original python.py pygment
+    further expanded and consolidated with the godot-docs lexer by Zackery R. Smith <zackery.smith82307@gmail.com> and Ste.
 """
 
-import re
-
-from pygments.lexer import (
-    RegexLexer,
-    include,
-    bygroups,
-    default,
-    words,
-    combined,
-)
+from pygments.lexer import RegexLexer, include, bygroups, words, combined
 from pygments.token import (
-    Text,
-    Comment,
-    Operator,
     Keyword,
+    Literal,
     Name,
+    Comment,
     String,
     Number,
-    Punctuation,
+    Operator,
     Whitespace,
+    Punctuation,
 )
 
 __all__ = ["GDScriptLexer"]
 
-line_re = re.compile(".*?\n")
-
 
 class GDScriptLexer(RegexLexer):
     """
-    For `GDScript source code <https://www.godotengine.org>`_.
+    For GDScript source code.
     """
 
     name = "GDScript"
-    aliases = ["gdscript", "gd"]
+    url = "https://www.godotengine.org"
+    aliases = ["gdscript", "gd", "gdscript2", "gd2"]
```

Comment:
I would remove the aliases we don't use:

```suggestion
    aliases = ["gdscript", "gd"]
```

This lexer should still do a good enough job with Godot 3.x GDScript code anyway, so the distinction isn't really relevant.

Reply:
ah yep, we're running the full own lexer here. in our case we had a conflict with the builtin pygments lexer forcing us to add an alias. will remove